test/messages.jl

```diff
@@ -107,7 +107,7 @@ using JSON
 
 #=
     @sync begin
-        io = BufferStream()
+        io = Base.BufferStream()
```

Comment:
That's annoying ;)
See: https://github.com/JuliaLang/julia/pull/25802#issuecomment-365448930